source/wp-content/themes/wporg-developer-2023/src/style/style.scss

```diff
@@ -120,20 +134,16 @@ body[class] {
 }
 
 @media (max-width: 781px) {
-	.wp-block-column[style*="border-right-"] {
-		border-bottom: 1px solid var(--wp--preset--color--light-grey-1);
-		border-right: none !important;
-	}
-
-	.wp-block-column[style*="border-right-"][style*="padding-right"] {
-		margin-left: calc(-1 * var(--wp--preset--spacing--edge-space)) !important;
-		margin-right: calc(-1 * var(--wp--preset--spacing--edge-space)) !important;
-		padding-left: var(--wp--preset--spacing--edge-space) !important;
-		padding-right: var(--wp--preset--spacing--edge-space) !important;
-	}
-
-	.wp-block-column[style*="border-right-"] + .wp-block-column[style*="padding-left"] {
-		padding-left: 0 !important;
+	.home-section-docs-api {
+		.wp-block-column {
+			padding-left: var(--wp--preset--spacing--edge-space) !important;
+			padding-right: var(--wp--preset--spacing--edge-space) !important;
+
+			&[style*="border-right-"] {
+				border-bottom: 1px solid var(--wp--preset--color--light-grey-1);
+				border-right: none !important;
+			}
+		}
```

Comment:
I'm unsure whether this pattern is used in other pages, so maybe the styles shouldn't be removed.

For this particular section, once the horizontal padding is removed from the columns to allow the layout to work in both directions, and added to the group wrapper instead, the border styling breaks, as it doesn't stretch edge to edge anymore.

IMO this changeset is simpler (no negative margin) and less brittle, but it's also less reusable. Open to discussion.